packages/react-devtools-extensions/firefox/test.js

```diff
@@ -10,13 +10,32 @@ const {argv} = require('yargs');
 const EXTENSION_PATH = resolve('./firefox/build/unpacked');
 const START_URL = argv.url || 'https://reactjs.org/';
 
+const ffVersion = process.env.WEB_EXT_FIREFOX;
```

Comment:
nit I generally like to avoid abbreviations in variable names like "ff". Totally not a big deal given the context of this change, but for consistency elsewhere I might rename this one before merging.